src/tabs/default.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  @include iui-tabs;
```

Comment:
`iui-tabs-default` is a modifier, so it shouldn't need to include base `iui-tabs` styles again.

Reply:
Oops, removed all instances of `@include iui-tabs;` from within modifiers.